delfin/drivers/fake_storage/__init__.py

```diff
@@ -80,7 +82,7 @@ def wait_random(low, high):
     def _wait(f, *a, **k):
         rd = random.randint(0, 100)
         secs = low + (high - low) * rd / 100
-        time.sleep(secs)
```

Comment:
what's the difference between `time.sleep` and `greenthread.sleep`

Reply:
```greenthread.sleep``` Yield control to another eligible coroutine until at least seconds have elapsed but with ```time.sleep```  call do not do same. It eventually keep multiple coroutine busy when we test performance metric collection with load test (register more than 100 storage).